Documentation/contributing.rst

```diff
@@ -622,6 +622,76 @@ common unit test frameworks.
 A good use case for using ``AfterAll`` method is to remove containers or pods
 that are needed for multiple ``Its`` in the given ``Context`` or ``Describe``.
 
+JustAfterEach
+^^^^^^^^^^^^^
+
+This method will run just after each test and before ``AfterFailed`` and
+``AfterEach``. The main reason of this method is to provide a mechanism to do
+some assertions, for a group of test, after the test finished without
+duplicating some the code. For example, for each test a new a dead lock check
```

Comment:
A good example of using a global ``JustAfterEach`` function is for deadlock detection, which checks the Cilium logs for deadlocks that may have occurred in the duration of the tests.